airflow/jobs/local_task_job.py

```diff
@@ -165,7 +160,7 @@ def handle_task_exit(self, return_code: int) -> None:
 
         if not self.task_instance.test_mode:
             if conf.getboolean('scheduler', 'schedule_after_task_execution', fallback=True):
-                self._run_mini_scheduler_on_child_tasks()
+                self.task_instance.schedule_downstream_tasks()
```

Comment:
Do we want to keep “mini scheduler” in the name?

Reply:
Ok. How about `self.task_instance.run_mini_scheduler_on_downstream_tasks`? The `mini scheduler` sounded to me like it's doing a lot

Reply:
Or we could just add “mini scheduler” to the docstring or comment; I’m just wondering whether we should keep the string somewhere so it’s easier for someone to find the relevant code when we talk about the mini scheduler.